### ISSUE TITLE
`linearGradient2` with degree direction value compiles to invalid CSS

### ISSUE THREAD

**Issue by maxhallinan** [rtfeldman/elm-css#335]
`linearGradient2` takes a `AngleOrDirection` as its first argument. It then [prepends this value](https://github.com/rtfeldman/elm-css/blob/master/src/Css.elm#L4339) with `"to"`. When that value is a degree instead of a direction, the result is invalid CSS.

Example:

```
linearGradient2  (deg 10) (stop <| hex "000") (stop <| hex "222") []
```

compiles to

```
linear-gradient(to 10deg, rgba(209, 237, 188, 0.5), rgba(209, 237, 188, 0.25))
```

Am I missing something? This library would benefit from an overview of the domain model. Reverse engineering the domain model from type annotations is not efficient for new users. Because I'm never sure that I understand the domain model, I'm not sure that I'm approaching this problem the correct way. If this is a legitimate bug, I'm happy to work on a fix. Wanted to make sure first.

**Comment by rtfeldman** [comment]
@maxhallinan Yeah, looks like a bug. I'd definitely appreciate a PR for a fix!

**Comment by maxhallinan** [comment]
Thanks for taking a look. I'll work on a fix.
